src/controllers/busStop.controllers.js

```diff
@@ -115,10 +115,21 @@ const updateBusStop = (req, res) => {
     ));
 };
 
+const searchBusStop = (req, res) => {
+  const { bstop } = req.query;
+  models.busStops
+    .findAll({ where: { busStopName: { [Op.like]: `%${bstop}%` } } })
+    .then((busStops) => res.status(200).json({ busStops }))
+    .catch(() => {
+      res.status(404).json({ status: 404, message: 'No Result found' });
+    });
+};
```

Comment:
Good work @Isaackomeza one thing to consider is that this search functionality is case sensitive so when a user searches `Nya`, they get the results as they are capitalised in the database, but when they search for `nya`, all in lowercase they don't get the results which I think would not be a good user experience. So, I think you can modify this to not be case-sensitive

Reply:
Thank you @maximeish for realizing this, 
I've been struggling with it since putting **bstop** to lowercase was not working alone, but now with sequelize **iLike** operator instead of **like** operator solved it.
Thank you!